src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
You've specified a primary font-family, which is great. However, it's a good practice to specify a fallback font in case the primary font doesn't load. This is typically a generic font-family like `sans-serif` or `serif`. In your case, you've already specified `sans-serif` as a fallback, which is good. But it would be better to specify one more, more specific fallback font before `sans-serif`. For example: `font-family: Roboto, Arial, sans-serif;`